packages/ibm-products/src/components/EmptyStates/EmptyState.test.js

```diff
@@ -147,7 +147,7 @@ describe(name, () => {
 
   it('should render the NoDataEmptyState', async () => {
     const { container, rerender } = render(
-      <NoDataEmptyState {...defaultProps} />
+      <NoDataEmptyState {...defaultProps} illustrationDescription="Test alt text" />
```

Comment:
Add `illustrationDescription` to the `defaultProps` object so that you don't have to explicitly pass it in for each test. Then you can just render the component like:
```jsx
render(<NoDataEmptyState {...defaultProps} />);
```

Reply:
@matthewgallo  , added the illustrationDescription to defaultProps